File: Python/URL/get_fragment_from_url.py
import urllib.parse


# Get the fragment from a given url.
def get_fragment_from_url(url):
    return urllib.parse.urlparse(url).fragment


def main():
    # Get the fragment from a given url.
    print(get_fragment_from_url("https://www.example.com#fragment"))


main()

File: TypeScript/URL/get-fragment-from-url.ts
import * as url from "url"

function main(): void {
    // Get the fragment from a given url.
    console.log(getFragmentFromURL("https://www.example.com#fragment"))
}

main()

// Get the fragment from a given url.
function getFragmentFromURL(givenURL: string): string | null {
    return new URL(givenURL).hash
}
